pkg/driver/videotest/dummy.go

```diff
@@ -129,10 +137,10 @@ func (d *dummy) VideoRecord(p prop.Media) (video.Reader, error) {
 		}, func() {}, nil
 	})
 
-	return r, nil
+	return r, d.state.Update(driver.StateRunning, func() error { return nil })
 }
 
-func (d dummy) Properties() []prop.Media {
+func (d *dummy) Properties() []prop.Media {
```

Comment:
why did this change to being a pointer?

Reply:
auto fix by IDE for mixed value and pointer receivers. Changed back.